### PR TITLE
Symfony 5 upgrade

### PR DESCRIPTION
**Pull request by alexandre-castelain**
I upgraded the questions to the Symfony 5 documentation

### CODE REVIEW

**Review by JeroenMoonen — APPROVED** [review]

**Comment by marcodepolo** [comment]
I've just made a pull request with the missing help sections and a few corrections, but I think my changes may need to be reviewed by someone with more symfony experience

**Comment by Calex92** [comment]
Thanks @marcodepolo, you did a great job man !

**Comment by marcodepolo** [comment]
You're welcome, glad to help and it was a great exercise for me to learn
symfony

**Comment by artsev** [comment]
What about merging this update? 👍

File: data/architecture.yml [hunk 1]
```diff
@@ -1,135 +1,168 @@
...
+            - {value: "APP--", correct: false}
+            - {value: "SYMFONY__", correct: false}
             - {value: "SF__",      correct: false}
             - {value: "SYMFONY-",  correct: false}
             - {value: "SF-",       correct: false}
+        help: |
+            'no answer found, however, for symfonycloud, the prefix is SYMFONY_ https://symfony.com/doc/current/cloud/cookbooks/env.html#symfonycloud-environment-variables'
     -
-        question: 'In Kernel class, which method do you need to redefine if you want to have multiple configuration files (config_dev.yml, config_test.yml, ...)?'
+        question: 'In Symfony\Component\HttpKernel\Kernel class, which method do you need to redefine if you want to have multiple configuration files (config_dev.yml, config_test.yml, ...)?'
```
Comment (artsev) [inline data/architecture.yml:118]: I think this question should be removed as it is completely obsolete in Symfony. The help text helps, but why the studying person should even bother answering it.

**Review by 20uf — APPROVED** [review]
Thank you for your contribution, it's really good!

For me everything is ok. If there are no comments from members of the organization, I will merge it shortly.

**Pull request merged**